tajo-storage/tajo-storage-hdfs/src/main/java/org/apache/tajo/storage/index/bst/BSTIndex.java

```diff
@@ -488,6 +497,18 @@ public BSTIndexReader(final Path fileName) throws IOException {
       open();
     }
 
+    public void hold() {
+      REFERENCE_UPDATER.compareAndSet(this, referenceNum, referenceNum + 1);
+    }
+
+    public void release() {
```

Comment:
It seems to need the closing after release. if you add the description, it will be better